index.html

```diff
@@ -56,83 +63,139 @@ <h2>LET'S CONNECT</h2>
+          
+                  <div class="card-container-1">
+                    <img src="images/snapshoot-portfolio-3.svg" alt="">
+                    <div class="card-content-1">
+                    <h3>multi-post stories</h3>
+                    <h4><span>Facebook</span>&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      full stack dev&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      2015
+                    </h4>
+                    <p>Exploring the future of media in facebook's first virtual Reality <br>
+                      app a place to discover and enjoy 360 points and videos on <br>
+                      Gear VR.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>Ruby on rails</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                </article>
+              </section>
+              <section class="about">
+                <div class="about-container">
+                  <div class="about-content">
+                  <h2>About <br>Myself</h2>
+                  <p>Hello I’m a software developer! I can help you build a product ,<br> feature
+                    or website Look through some of my work and experience!<br> If you like what
+                    you see and have a project you need coded, <br> don’t hestiate to contact me.</p>
+                  <p class="connect">let's connect</p>
+                  <ul class="social">
+                    <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/linkedin.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/medium.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+                    <li><a href="#"><img src="images/angellist.svg" alt=""></a></li>
+                  </ul>
+                  <button class="about-btn">get my resume</button>
+                  </div>
+                  <div class="btn-icon-language">
+                  <h3><span>languages</span><img src="images/arr-down.svg" alt=""></h3>
+                  <ul class="stack">
+                    <li><img src="images/js.svg" alt="">&nbsp;&nbsp;&nbsp;<span>JavaScript</span></li>
+                    <li><img src="images/html.svg" alt="">&nbsp;&nbsp;&nbsp;<span>HTML</span></li>
+                    <li><img src="images/css.svg" alt="">&nbsp;&nbsp;&nbsp;<span>CSS</span></li>
+                  </ul>
+                  <h3><span>frameworks</span><img src="images/arr-right.svg" alt=""></h3>
+                  <h3 class="skills"><span>skills</span><img src="images/arr-right.svg" alt=""></h3>
+                  </div>
+                </div>
```

Comment:
- [ ] [OPTIONAL] Also, your about section in the desktop version doesn't look like the one in the Figma design, I suggest sticking to the Figma design and making some changes so your contact form will be as one in the Figma design 

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/165811351-f32eacc3-b15e-473e-acf2-d0061ad7f540.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/165811441-15ba8ae2-4829-4352-9b35-c5605caaed0b.png)